hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2325,6 +2325,67 @@ func (r *HostedClusterReconciler) reconcileAutoscaler(ctx context.Context, creat
 	return autoscaler.ReconcileAutoscaler(ctx, r.Client, hcp, clusterAutoscalerImage, utilitiesImage, createOrUpdate, r.SetDefaultSecurityContext, config.MutatingOwnerRefFromHCP(hcp, releaseVersion))
 }
 
+func (r *HostedClusterReconciler) reconcileETCDMemberRecovery(ctx context.Context, hcluster *hyperv1.HostedCluster, createOrUpdate upsert.CreateOrUpdateFN) error {
```

Comment:
Why not do this in the CPO?  we really don't want the HO deploying things to the HCP beyond CAPI and CPO.  Also, this function isn't called anywhere (cause of the `verify` failure)